<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubsonicSettingsPage</class>
 <widget class="QWidget" name="SubsonicSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subsonic</string>
  </property>
  <layout class="QVBoxLayout" name="layout_subsonicsettingspage">
   <item>
    <widget class="QCheckBox" name="enable">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="server_group">
     <property name="title">
      <string>Server URL</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="layout_server">
        <item>
         <widget class="QLineEdit" name="server_url"/>
        </item>
        <item>
         <spacer name="spacer_server">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="credential_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QFormLayout" name="layout_credential_group">
      <item row="1" column="0">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="username"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_auth_method">
     <property name="title">
      <string>Authentication method:</string>
     </property>
     <layout class="QHBoxLayout" name="layout_auth_method">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="auth_method_hex">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="auth_method_md5">
        <property name="text">
         <string>MD5 token (Recommended)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_auth">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_preferences">
     <property name="title">
      <string>Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkbox_http2">
        <property name="text">
         <string>Use HTTP/2 when possible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_verify_certificate">
        <property name="text">
         <string>Verify server certificate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_download_album_covers">
        <property name="text">
         <string>Download album covers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_use_album_id_for_album_covers">
        <property name="text">
         <string>Use album ID for album covers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_server_scrobbling">
        <property name="text">
         <string>Server-side scrobbling</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_tools">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="button_test">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_deletesongs">
        <property name="text">
         <string>Delete songs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_middle">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_bottom">
     <item>
      <spacer name="spacer_bottom">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_subsonic">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../data/icons.qrc">:/icons/64x64/subsonic.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enable</tabstop>
  <tabstop>server_url</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>checkbox_verify_certificate</tabstop>
  <tabstop>checkbox_download_album_covers</tabstop>
 </tabstops>
 <resources>
  <include location="../../data/data.qrc"/>
  <include location="../../data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
